v0/sources/refiner/identifyMapping.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "sourceKeys": "email",
+    "destKeys": "context.traits.email"
+  },
+  {
+    "sourceKeys": "remote_id",
+    "destKeys": "userId"
+  },
+  {
+    "sourceKeys": "attributes",
+    "destKeys": "context.traits"
+  },
+  {
+    "sourceKeys": "response.data",
+    "destKeys": "traits"
+  },
+  {
+    "sourceKeys": "form.uuid",
+    "destKeys": "context.formUuid"
+  },
+  {
+    "sourceKeys": "form.name",
```

Comment:
same for this, why we are not mapping them to context.traits but rather in context object directly